examples/asr/librispeech_ctc_decoder/inference.py

```diff
@@ -0,0 +1,126 @@
+import argparse
+from typing import Optional
+
+import torch
+import torchaudio
+from torchaudio.prototype.ctc_decoder import kenlm_lexicon_decoder
+
+
+bundle_map = {
+    "FT_10M": torchaudio.pipelines.WAV2VEC2_ASR_BASE_10M,
+    "FT_100H": torchaudio.pipelines.WAV2VEC2_ASR_BASE_100H,
+    "FT_960H": torchaudio.pipelines.WAV2VEC2_ASR_BASE_960H,
```

Comment:
Instead of defining a closed set of the supported models, making it open will reduce the maintenance cost.

For example, we can use `bundle = getattr(torchaudio.pipelines, 'model')`.